client/src/components/VictoryBox.jsx

```diff
@@ -27,7 +27,7 @@ export const VictoryBox = (props) => {
   }, []);
 
   return (
-    <div className='flex flex-col bg-white rounded-t-md'>
+    <div className='flex flex-col bg-white border-b-lg'>
```

Comment:
```suggestion
    <div className='flex flex-col bg-white sm:rounded-b-lg'>
```
legg kun på borders når man er på større skjermer (på mindre enheter når lengen ut i kantene på skjermen)